R/Prediction.R

```diff
@@ -195,6 +195,57 @@ makePrediction.CostSensTaskDesc = function(task.desc, row.names, id, truth, pred
 }
 
 #' @export
+makePrediction.ForecastRegrTaskDesc = function(task.desc, row.names, id, truth, predict.type,
+                                               predict.threshold = NULL, y, time, error = NA_character_, dump = NULL) {
+  data = namedList(c("id", "truth", "response", "se"))
+  if (any(inherits(y, "matrix")))
+    size.y = nrow(y)
+  else
+    size.y = length(y)
+
+  # This will only happen when there is a task with no subset
+  #  aka, we predict future values and have to get their times
+  if (length(truth) > size.y) {
```

Comment:
Yes
I wanted to include actual dates for the forecast. i.e. if I am at January 2nd, 2015 and forecast five periods ahead I want my predictions for be for the 3rd, 4th, 5th, 6th, 7th. So I have to do this stuff to figure out the dates for the forecast.

We simply take the median of the task data's dates as that is reasonably accurate for short periods.